pkg/kstatus/observe/event/event.go

```diff
@@ -0,0 +1,125 @@
+package event
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/cli-utils/pkg/kstatus/status"
+	"sigs.k8s.io/cli-utils/pkg/kstatus/wait"
+)
+
+// EventType is the type that describes the type of an Event that is passed back to the caller
+// as resources in the cluster are being observed.
+type EventType string
```

Comment:
if u want, you can use [stringer](https://blog.golang.org/generate), a utility Rob Pike wrote to convert int enums to strings for printing

E.g. this
https://github.com/kubernetes-sigs/kustomize/blob/master/api/types/loadrestrictions.go
generates this
https://github.com/kubernetes-sigs/kustomize/blob/master/api/types/loadrestrictions_string.go

i suppose the same would apply to pkg/kstatus/status/status.go   

up to you.  keeps channel content lower, and avoids spelling typos by relying on the iota mechanism for uniqueness